src/scss/overview/_search-form.scss

```diff
@@ -1,36 +1,33 @@
-.registry__form {
-
-	&-search {
-		position: relative;
-	}
+.registry__form-search {
+	position: relative;
+}
 
-	&-search-loading {
-		@include oLoading();
-	}
-	&-search-loading {
-		@include oLoadingColor('dark');
-		width: 10px;
-		height: 10px;
-		border-width: 2px;
-		position: absolute;
-		right: 12px;
-		bottom: 12px;
-		display: none;
-	}
+.registry__form-search-loading {
+	@include oLoadingContent($opts: (
+		'theme': 'dark'
+	));
+}
 
-	&--loading &-search-loading {
-		display: block;
-	}
+// registry loading class is defined twice intentionally.
+// o-loading mixins create a new selector
+// after the one they are used in, to output
+// rules after keyframes
```

Comment:
I don't understand what this is trying to explain 😔

Reply:
`oLoadingContent` outputs css at the root using `@at-root` and creates a new selector.
So custom styles that follow `oLoadingContent` do not come after the mixin output 😞 
https://github.com/Financial-Times/o-loading/blob/082a9c5e35f1fa61b3adce755b1df5193b72e0c8/src/scss/_mixins.scss#L79

Does that make more sense as a comment?

Reply:
That does make more sense 👍